js/views/modals/Settings/Settings.js

```diff
@@ -198,10 +197,6 @@ export default class extends BaseModal {
       this.selectTab(this.$('.js-tab[data-tab="General"]'));
     });
 
-    installRichEditor(this.$('#settingsAbout'), {
-      topLevelClass: 'clrBr',
-    });
-
```

Comment:
The rich text editor was moved to the page tab view, it has to be in the view that matches the template, otherwise the element can't be selected because the view can't access it.